sdk/python/testvectors/__main__.py

```diff
@@ -33,44 +36,225 @@ def get_modules(network_name):
 	return modules
 
 
-def prepare_transaction_test_cases(network_name):
-	filepath = Path(os.environ.get('SCHEMAS_PATH', '.')) / network_name / 'transactions' / 'transactions.yaml'
-	return read_test_vectors_file(filepath)
+def to_hex_string(buffer):
+	return hexlify(buffer).decode('utf8').upper()
+
+
+def clone_descriptor(descriptor):
+	# note, this makes shallow clone
+	cloned_descriptor = {}
+	for key, value in descriptor.items():
+		if key in ['deadline', 'signature', 'fee']:
+			continue
+
+		cloned_descriptor[key] = value
+
+	return cloned_descriptor
+
+
+class SymbolHelper:
+	AGREGATE_SCHEMA_NAME = 'AggregateBondedTransaction'
+	Signature = sc.Signature
+
+	def __init__(self, network_name):
+		self.facade = SymbolFacade(network_name)
+
+	def set_common_fields(self, descriptor, test_name):
+		descriptor['signer_public_key'] = sha3.sha3_256(test_name.encode('utf8')).hexdigest()
```

Comment:
i guess this doesn't need to be wrapped in PublicKey?

Reply:
nope (although could)